eth-bridge/contracts/README.md

```diff
@@ -56,18 +56,19 @@ Following security features are implemented in the contract:
   recipient etc)
 * bridge can be stopped by a watcher at any time
 * bridge enforces rate-limit on mints
-* bridge limits how many wrapped tokens can in circulation
+* bridge enforces minimum transfer amount
+* bridge limits how many wrapped tokens can be in circulation
 * there's a delay between approval of mint and actually allowing it
 
 ## Official deployments
 
 Sepolia testnet (meant to be used with Liberland Bastiat testnet):
 
 ```
-LDN ERC-20 token: 0x195E89cdE9475e18DB594A24FC2954235d7F4e92
+LDN ERC-20 token: 0x0f018D7e0B8f5D5cCc88c0B23d931AaAA13B0C42
```

Comment:
why

Reply:
Ah, sorry, this should be part of https://github.com/liberland/liberland_substrate/pull/304/files .
Will clean this from this PR.
As to why: old token contract required explicit allowance granted to be used with bridge by each token holder. New version allows bridge use automatically.

Reply:
Removed these changes from this PR.